include/Reader/reader.h

```diff
@@ -47,12 +47,7 @@
 // Debugging
 // -----------------------------------------------------------------
 
-#define ASSERTM(Predicate, Message)                                            \
-  do {                                                                         \
-    if (!(Predicate)) {                                                        \
-      ReaderBase::debugError(__FILE__, __LINE__, Message);                     \
```

Comment:
I believe this was the only caller of ReaderBase::debugError, so it could be removed now.